src/java/org/apache/cassandra/service/accord/txn/TxnWrite.java

```diff
@@ -263,7 +263,7 @@ public Update complete(AccordUpdateParameters parameters)
             if (!staticRow.isEmpty())
                 updateBuilder.add(staticRow);
 
-            Row existing = !baseUpdate.isEmpty() ? Iterables.getOnlyElement(baseUpdate) : null;
+            Row existing = baseUpdate.hasRows() ? Iterables.getOnlyElement(baseUpdate) : null;
```

Comment:
there are 3 things that define "not empty":

1) has row
2) has static row
3) has liveness info

so for our use case we need `hasRow` as we are touching the row

Reply:
Yeah, no reason we should have looked at the static row, as it's already accounted for above...